src/content/reference/react-dom/server/renderToStaticMarkup.md

```diff
@@ -16,62 +16,62 @@ const html = renderToStaticMarkup(reactNode)
 
 ---
 
-## Reference {/*reference*/}
+## Referensi {/*reference*/}
 
 ### `renderToStaticMarkup(reactNode)` {/*rendertostaticmarkup*/}
 
-On the server, call `renderToStaticMarkup` to render your app to HTML.
+Pada *server*, panggil fungsi `renderToStaticMarkup` untuk me-*render* aplikasi Anda sebagai HTML.
```

Comment:
Opini: Awalnya saya berpikir untuk menggunakan `penyedia layanan`, tetapi saya merasa bahwa *server* merupakan bentuk yang lebih mudah dipahami secara garis besar.

Reply:
Yes keep it as `server`

We'll also have `*server-side rendering*` or `proses render secara *server-side*` too